cfme/services/requests.py

```diff
@@ -180,7 +181,7 @@ def wait_for_request(cells, partial_check=False):
          The matching :py:class:`cfme.web_ui.Table.Row` if found, ``False`` otherwise.
     """
     row = find_request(cells, partial_check)
-    if row.request_state.text in REQUEST_FINISHED_STATES:
+    if row.request_state.text in REQUEST_FINISHED_STATES or REQUEST_PENDING_STATE:
```

Comment:
**REQUIRED**
It doesn't work like that :)
```python
if row.request_state.text in REQUEST_FINISHED_STATES or row.request_state.text in REQUEST_PENDING_STATE:
```
Bu I think it's a bit better will be:
```python
if row.request_state.text in list(REQUEST_PENDING_STATE) + list(REQUEST_FINISHED_STATES):
```